UIKitMovie/UIKitMovie/View/DetailMovieViewController.swift

```diff
@@ -78,8 +87,27 @@ final class FilmViewController: UIViewController {
         filmImageView.contentMode = .scaleToFill
         filmImageView.layer.borderWidth = 1
         filmImageView.layer.borderColor = Constants.yellowCustomColor?.cgColor
-        guard let imageName = film?.backdropPath else { return }
-        filmImageView.updateImageName(URLAddres: imageName)
+        guard let imagePath = detailMovieViewModel?.film?.backdropPath else { return }
+        fetchImage(url: imagePath)
+    }
+
+    private func fetchImage(url: String) {
```

Comment:
```suggestion
    private func fetchImage(url: String) {
```
этот метод в viewMоdel должен быть